tslint.json

```diff
@@ -21,7 +21,8 @@
       "no-console": false,
       "no-var-requires": false,
       "comment-format": false,
-      "ordered-imports": false
+      "ordered-imports": false,
+      "no-unused-expression": false // Messes with the tests when using expect or assert
```

Comment:
I don't think it's good solution to put this here. Maybe better to define it only for tests and ideally don't use it at all.